src/unix/lwt_unix.cppo.mli

```diff
@@ -121,7 +121,22 @@ val yield : unit -> unit Lwt.t
 
 val auto_yield : float -> (unit -> unit Lwt.t)
   (** [auto_yield timeout] returns a function [f] that will yield
-      every [timeout] seconds. *)
+      every [timeout] seconds.
+      Consider a function f defined with let f = auto_yield timeout. If you run
+      multiple times this function, it will behave like either Lwt.return () or
+      Lwt_unix.yield depending on the elapsed time.
+
+      t = 0
+        f acts like Lwt.return ()
+      t <= timeout
+        f acts like Lwt.return ()
+      t >= timeout
+        f acts like Lwt_unix.yield ()
+
+      if f is re-ran after it has yielded, f will act like if t = 0.
```

Comment:
"re-r*u*n" for the subjunctive.

The comment about `t = 0` makes it seem like it will *always* behave like `Lwt.return` after the first `yield`, because it does not say that the timeout is restarted, and it says above that when `t = 0`, `f ()` is `Lwt.return ()`.